packages/shared/src/components/auth/AuthExtensionBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import { BottomBannerContainer } from '../banners';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { AuthTriggers } from '../../lib/auth';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { GetExtensionButton } from '../buttons/GetExtensionButton';
+import { cloudinaryAuthBannerBackground as bg } from '../../lib/image';
+import { Image } from '../image/Image';
+
+const laptopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1440/f_auto/');
+const desktopBg = bg.replace('/f_auto', '/c_auto,g_center,w_1920/f_auto/');
+
+export const AuthExtensionBanner = (): ReactElement => {
+  const { showLogin } = useAuthContext();
+
+  return (
+    <BottomBannerContainer className="gap-10 border-t border-none border-accent-cabbage-default py-10 shadow-3">
+      <Image
+        className="absolute left-0 top-0 -z-1 h-full w-full"
+        src={bg}
+        srcSet={`${laptopBg} 1440w, ${desktopBg} 1920w, ${bg} 2880w`}
```

Comment:
When using srcSet we need to manually add onError as now the default won't trigger.

Reply:
Still personally think f_auto is enough for these edge-cases though.

Reply:
@rebelchris Ah, thanks for the bit about onError, will add.

As far as the crops go, initially I agreed, but @ilasw showed [here](https://github.com/dailydotdev/apps/pull/3847#discussion_r1850657413) that the difference was quite noticable. 1440 and 1920 save about 7 and 8kb respectively.

Although, it would've perhaps been okay to only go with the 1920 version of the image and just have that as a background.

Reply:
Yeah just feels like it adds so much overhead and saving for very little endpoints.
But not really blocking

Reply:
> When using srcSet we need to manually add onError as now the default won't trigger.

Why do we need the onError? Here we don't have the change to have undefined/wrong links for this link 👀

Reply:
Yeah I guess these are defined by us.
So guess it's ok for this case maybe